tracardi/service/storage/drivers/elastic/tag.py

```diff
@@ -0,0 +1,47 @@
+from tracardi.service.storage.factory import storage_manager
+from typing import List
+
+
+async def get_document_by_event_type(event_type):
+    return await storage_manager("event-tags").load_by(field="type", value=event_type)
+
+
+async def add_tags(event_type: str, tags: List[str]):
+    search_result = await get_document_by_event_type(event_type=event_type)
+    if search_result.dict()["result"]:
+        tags.extend(search_result.dict()["result"][0]["tags"])
+        tags = list(set(tags))
+        return 1, await storage_manager("event-tags").upsert({
+            "_id": search_result.dict()["result"][0]["id"],
+            "type": search_result.dict()["result"][0]["type"],
+            "tags": tags
+        })
+    else:
+        return 0, await storage_manager("event-tags").upsert({
```

Comment:
What 0 and 1 means in return. Maybe you could add some documentations so users can understand it?

Reply:
I saw in tracardi API that this is the number of updated records. What happens if there is an error. Elastic returns the number of updated records.